### ISSUE TITLE
Filter library by series name

### ISSUE THREAD

**Issue by madscientist16** [Acclorite/book-story#105]
**Please, follow _all_ of these checks (Issues that do not will be canceled):**
- [x] This feature is not implemented and does not have opened/closed Issues.
- [x] I propose ONLY one feature at a time. One feature - one Issue.
- [x] I use English. This repository's language is English.
- [x] I properly filled the template below(not necessarily all parts, but at least first two).
- [x] I did not use any placeholder such as [Feature] or [Bug] in the title of the Issue.

**Why you want this feature to be implemented? Please describe.**

I would like to be able to easily see which books are from which series.

**Describe the feature you'd like to be implemented**

I'm not sure honestly. Maybe add a seperate page for each series like it is on Mihon like in this screenshot. 
![1000030520](https://github.com/user-attachments/assets/e5aabf37-4975-4514-bfd0-414615149091)
Instead of having one main cover art like in Mihon we could have cover arts shown for each book in the series along with their description replacing the chapters list of Mihon in the screenshot.

**Describe alternatives you've considered**

Issue #79 is about adding custom categories and it could be used as a way to seperate books from different series as well.

**Additional context**

**Comment by yslflow** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by madscientist16** [comment]
Hello, can the above comment be deleted. They linked to a webpage asking to run some weird powershell script. The same comment is in a couple of other issues too.

**Comment by Acclorite** [comment]
Yes, blocked. Please, do not click any links by other users in Issues(or triple check them).

**Comment by IDtheTarget** [comment]
I would like this feature as well. I have many series in Calibre, with the Series Title and the Number entered. Calibre puts them as:
```
<meta name="calibre:series" content="Heinlein Opus List" />
<meta name="calibre:series_index" content="136.0" />
```
and stores the info in the content.opf file in the epub.

In Moon+ Reader, when you are sorting or searching for your books, you can choose a series and see the books therein.

Thank you for this app! it's *almost* perfect!
